data/map.txt

```diff
@@ -26762,7 +26762,7 @@ planet Ahr
 	outfitter "Coalition Advanced"
 
 planet Aksaray
-	attributes moon "requires: inaccessible" uninhabited
```

Comment:
This PR still seems to contain test-data (here and below).

Reply:
Well that's the thing this PR fixes precisely so may as well fix it directly instead of forcing Zitchas to do another PR right afterwards.

Reply:
I'm fine with doing a PR to do this if necessary; but I am also great with it being handled in here.

Reply:
It's helpful to have them in this PR for easy testing.

Reply:
Yes, if this change is intentional and desired, then I agree to keep it as part of this PR. (And I should have read the PR text more clearly, it was also properly stated there. :) )